### PR TITLE
Tuner - new package

### PR DESCRIPTION
**Pull request by MechDR**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by misuchiru03** [comment]
I noticed when it finally built, the file names it provides are gross: `com.github.louis77.tuner`, and all other files where the binary name of `tuner` would suffice follows this naming.

**Comment by classabbyamp** [comment]
the inverse-DNS naming scheme is fine. there's no need to change it

File: srcpkgs/tuner/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'tuner'
+pkgname=tuner
+version=1.5.1
+revision=1
+build_style=meson
+hostmakedepends="gettext pkg-config vala"
```
Comment (tranzystorek-io) [inline srcpkgs/tuner/template:6]: Seems `glib-devel` is needed as `hostmakedepends` as well to get the `glib-compile-resources` on cross targets

**Comment by tranzystorek-io** [comment]
Also these commits should be squashed together into one `New package: tuner-1.5.1`

File: srcpkgs/tuner/template (hunk 1 above)
Comment (MechDR) [reply srcpkgs/tuner/template]: I'm sorry, I'm kinda new to all this... is what I've done OK? :-\\

**Comment by MechDR** [comment]
Are these commits OK?

Also, it states the version is 1.5.0, but it's actually 1.5.1, the author just forgot to change the version number in the About.

File: srcpkgs/tuner/template (hunk 1 above)
Comment (tranzystorek-io) [reply srcpkgs/tuner/template]: Should be good, with a few remarks:

- any blank lines are unnecessary
- we usually indent wrapped lines like the `makedepends` with only one space

As mentioned below we do one commit per package, so you need to squash all these. In the future, you can use `git commit --amend` to add changes without introducing new commits.

Comment (MechDR) [reply srcpkgs/tuner/template]: > * any blank lines are unnecessary

I believe I had only one blank line at the end... POSIX standard I believe, everything ends with a new line... please correct me if I'm wrong :).

> * we usually indent wrapped lines like the `makedepends` with only one space

As in not do a bunch of spaces to align the indent? OK, no problem, feel free to correct the template and I will remember this for future commits ;).

> As mentioned below we do one commit per package, so you need to squash all these. In the future, you can use `git commit --amend` to add changes without introducing new commits.

I still haven't connected my main rig with GitHub, I'm still working on the web interface :-\\... I know, I'll get on it as soon as possible.

Now, how do I squash the commits :). Might be a stupid question, I know, but as I said, I'm kinda new :).

Comment (tranzystorek-io) [reply srcpkgs/tuner/template]: > Now, how do I squash the commits

See "Squashing commits" in https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History, should be pretty clearly described (beware, `git rebase -i` sends you to the default editor, which might be vim on linux 😛)

Comment (paper42) [reply srcpkgs/tuner/template]: That's easy to solve, don't use the web ui.

File: srcpkgs/tuner/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'tuner'
+pkgname=tuner
+version=1.5.1
+revision=1
+build_style=meson
+hostmakedepends="glib-devel gettext pkg-config vala"
+makedepends="geoclue2-devel geocode-glib-devel geocode-glib-libsoup2-devel
```
Comment (paper42) [inline srcpkgs/tuner/template:7]: you are mixing libsoup2 and libsoup3 versions of geocode-glib here, remove the libsoup3 one (geocode-glib-devel) or switch the whole package to libsoup3 if possible.

Comment (MechDR) [reply srcpkgs/tuner/template]: Is this OK?

Comment (paper42) [reply srcpkgs/tuner/template]: Yes, do you know if there is a plan to support libsoup3?

Comment (MechDR) [reply srcpkgs/tuner/template]: > Yes, do you know if there is a plan to support libsoup3?

No, I'm not in contact with the author.

**Pull request closed without merging**